StoreApp.BLL/Services/BrandService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using AutoMapper;
+using StoreApp.BLL.DTO;
+using StoreApp.BLL.Interfaces;
+using StoreApp.DAL.Entities;
+using StoreApp.DAL.Intefaces;
+using System.Collections.Generic;
+
+namespace StoreApp.BLL.Services
+{
+    public class BrandService : IBrandService
+    {
+        IUnitOfWork DataBase { get; set; }
+        public BrandService(IUnitOfWork uof)
+        {
+            DataBase = uof;
+        }
```

Comment:
The methods must be splitted by using indents

Reply:
fixed